pallets/vault-registry/src/tests.rs

```diff
@@ -1094,9 +1096,18 @@ fn get_required_collateral_for_wrapped_with_threshold_succeeds() {
 				.unwrap();
 
 			// Check that the amount we found is indeed the lowest amount that is sufficient for
-			// `xlm`
-			assert!(max_xlm_for_min_collateral.amount() >= xlm);
-			assert!(max_xlm_for_below_min_collateral.amount() < xlm);
+			// `wrapped`
+			// We have to be a little generous here, because the conversion from collateral to
+			// wrapped may be rounded. So we give 10^(wrapped_decimals - collateral_decimals) as
+			// tolerance.
+			let tolerance = 10u128.pow(
+				<Test as oracle::Config>::DecimalsLookup::decimals(DEFAULT_WRAPPED_CURRENCY)
+					.saturating_sub(<Test as oracle::Config>::DecimalsLookup::decimals(
+						DEFAULT_COLLATERAL_CURRENCY,
+					)),
+			);
+			assert!(max_wrapped_for_min_collateral.amount() + tolerance >= wrapped_amount);
+			assert!(max_wrapped_for_below_min_collateral.amount() < wrapped_amount);
```

Comment:
Why don't we need the tolerance in the second assertion?

Reply:
Good question. From empiric testing I can tell you that it is not problematic and the bound holds even without the tolerance. But it would make sense to add the tolerance both ways.

Reply:
I added that tolerance also to the other check now